Add Simplified Chinese translation of a_tour_of_sage and tutorial

fisheryv opened the pull request:
Sagemath has long been without Simplified Chinese translations for its tutorials, which I believe significantly hinders the use and spread of SageMath. Therefore, I've decided to initiate a project to translate all tutorials into Simplified Chinese, striving for the translations to be faithful, fluent, and elegant. Here is the Simplified Chinese translation of a_tour_of_sage and sage_tutorial.

> I think we can closed the issue #9577 and #12559 .

### :memo: Checklist

<!-- Put an `x` in all the boxes that apply. -->

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [x] I have updated the documentation and checked the documentation preview.

fisheryv committed:
Add Simplified Chinese translation of a-tour-of-sage

fisheryv committed:
Add Simplified Chinese translation of a-tour-of-sage

fisheryv committed:
Update the LaTeX configuration of the document of a-tour-of-sage

fisheryv committed:
Merge branch 'sagemath:develop' into develop

fisheryv committed:
1. Update `source_view_link` and `source_edit_link`
2. Add new documents to website/root_index.html,

fisheryv committed:
Merge branch 'develop' of https://github.com/fisheryv/sage into develop

fisheryv committed:
1. Add Simplified Chinese translation of tutorial

fisheryv committed:
Fix some trivial formatting errors.

fisheryv committed:
Add Chinese language support to texlive

dimpase reviewed (approved):
Good idea. Now, let's see how it runs on the CI. @tscrim @roed314 - please approve the CI runs

fisheryv committed:
Merge remote-tracking branch 'upstream/develop' into develop

# Conflicts:
#	CITATION.cff
#	VERSION.txt
#	build/pkgs/configure/checksums.ini
#	build/pkgs/configure/package-version.txt
#	build/pkgs/sage_conf/version_requirements.txt
#	build/pkgs/sage_docbuild/version_requirements.txt
#	build/pkgs/sage_setup/version_requirements.txt
#	build/pkgs/sage_sws2rst/version_requirements.txt
#	build/pkgs/sagelib/version_requirements.txt
#	build/pkgs/sagemath_bliss/version_requirements.txt
#	build/pkgs/sagemath_categories/version_requirements.txt
#	build/pkgs/sagemath_coxeter3/version_requirements.txt
#	build/pkgs/sagemath_environment/version_requirements.txt
#	build/pkgs/sagemath_mcqd/version_requirements.txt
#	build/pkgs/sagemath_meataxe/version_requirements.txt
#	build/pkgs/sagemath_objects/version_requirements.txt
#	build/pkgs/sagemath_repl/version_requirements.txt
#	build/pkgs/sagemath_sirocco/version_requirements.txt
#	build/pkgs/sagemath_tdlib/version_requirements.txt
#	pkgs/sage-conf/VERSION.txt
#	pkgs/sage-conf_conda/VERSION.txt
#	pkgs/sage-conf_pypi/VERSION.txt
#	pkgs/sage-docbuild/VERSION.txt
#	pkgs/sage-setup/VERSION.txt
#	pkgs/sage-sws2rst/VERSION.txt
#	pkgs/sagemath-bliss/VERSION.txt
#	pkgs/sagemath-categories/VERSION.txt
#	pkgs/sagemath-coxeter3/VERSION.txt
#	pkgs/sagemath-environment/VERSION.txt
#	pkgs/sagemath-mcqd/VERSION.txt
#	pkgs/sagemath-meataxe/VERSION.txt
#	pkgs/sagemath-objects/VERSION.txt
#	pkgs/sagemath-repl/VERSION.txt
#	pkgs/sagemath-sirocco/VERSION.txt
#	pkgs/sagemath-tdlib/VERSION.txt
#	src/VERSION.txt
#	src/bin/sage-version.sh
#	src/sage/version.py

fisheryv committed:
Revert all the translated comments in the code back to their original language.

fisheryv committed:
Add the `tools/` directory to docker image to resolve the error reported during the CI Linux incremental process:
```
make[2]: ./tools/test-git-no-uncommitted-changes: No such file or directory
make[2]: *** [Makefile:257: test-git-no-uncommitted-changes] Error 127
```

fisheryv committed:
Merge branch 'develop' into develop
